### DISCUSSION TITLE
What does `tick` mean in slider property?

### DISCUSSION THREAD

**Discussion by eliasfloreteng** [rocksdanister/lively#522] [Q&A] [answered]
I am working on my [audio visualizer](https://github.com/elias123tre/lively-audio-visualizer) for lively and I am struggling with the tick option when configuring a slider in `LivelyProperties.json`.
My first question is what the tick option actually does and what i should change mine to. I tried a couple of different values and I observed no difference whatsoever.
And I am experiencing some strange behavior with float numbers for the slider. When I set the step value to a float the number i receive in `livelyPropertyListener` is the float as an integer without the decimal. For example if i set it to 4.5 it is received as 45. It works when first reloading the wallpaper (changing to another one and back) but not when changing the setting or resting to default.
I think it would be useful to also add this information to the wiki because I didn't find it there :)

Love the Lively program and your work; developing wallpapers for this community is such a joy!

**Comment by rocksdanister** [comment, marked as answer]
>My first question is what the tick option actually does and what i should change mine to. I tried a couple of different values and I observed no difference whatsoever.

tick is no longer used.

Are you using installer or store version of Lively?

**Reply by eliasfloreteng** [reply]
I am using Github releases version, aka installer

**Reply by rocksdanister** [reply]
There are some implementation differences between them.. from first look the github version looks fine to me.. will take a closer look later 🤔

**Installer**
https://github.com/rocksdanister/lively/blob/e9dab03513fc1b54572bf738f6b4ea4de0eb59f9/src/livelywpf/livelywpf/Cef/LivelyProperties/Page/LivelyPropertiesView.xaml.cs#L279-L317

API Doc:
https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.slider?view=winrt-19041

**Store**
https://github.com/rocksdanister/lively/blob/9c2c56bb317bb707b4a5f7229f9f384185e299b7/src/livelywpf/livelywpf/Cef/LivelyProperties/Page/LivelyPropertiesView.xaml.cs#L108-L138

API Doc:
https://docs.microsoft.com/en-us/dotnet/api/system.windows.controls.slider?view=net-5.0

**Reply by eliasfloreteng** [reply]
Ok, I will try to reinstall and see if it solves the issue. The weird thing is that it works when reloading the theme and the previous setting is a float, but doesn't work when I change it after that.

**Reply by rocksdanister** [reply]
Don't reinstall, reinstalling wont change any wallpaper related data..

**Reply by eliasfloreteng** [reply]
I see. I am not that familiar with UWP but the issue could be caused by `xamlSlider.ValueChanged` because it stops to work only when changed

**Reply by rocksdanister** [reply]
>When I set the step value to a float the number i receive in livelyPropertyListener is the float as an integer without the decimal. For example if i set it to 4.5 it is received as 45. It works when first reloading the wallpaper (changing to another one and back) but not when changing the setting or resting to default.

Can you describe in detail how to reproduce this and also share the project file, so far I cant recreate this error on my system.
Did you make sure the value field initially set is also float? the step conversion only takes place when slider is moved.

**Reply by eliasfloreteng** [reply]
I can share my project file but shouldn't I open a more detailed issue so we can track the progress there? If so then I can create it later today.

**Reply by rocksdanister** [reply]
Ok, I plan to release next update before end of this month.. if you can create the issue early I can include the patch with the next update.

**Reply by eliasfloreteng** [reply]
#525
